### PR TITLE
Changes how reagent containers transfer reagents

### PR DESCRIPTION
**Pull request by Chronitonity**
Reagent containers (Like beakers) now use attack_self instead of a verb to change how much units they transfer.

In normal speak, this means that clicking them changes how much they transfer instead of needing to right click it.

Reagent Dispensers like water/fueltanks now use the Containers transfer amount instead of their own amount.

Gameplay wise, this means you can fill bottles/glasses/bowls/ect faster and easier, and gets rid of a terrible right click verb.

### CODE REVIEW

File: html/changelogs/Chronitonity-ChemBuff.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   spellcheck (typo fixes)
+#   experiment
+#   tgs (TG-ported fixes?)
+#################################
+
+# Your name.  
+author: N3X15
```
Comment (Carbonhell) [inline html/changelogs/Chronitonity-ChemBuff.yml]: you forgot to add your name

File: html/changelogs/Chronitonity-ChemBuff.yml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - tweak: "All reagent containers (beakers, glasses, buckets ect) now is done by clicking them in your hands, just like how spray bottles work.
+  - tweak: ""Reagent dispensers like water or fueltanks no longer only transfer 10u, they use the containers transfer amount.
```
Comment (Carbonhell) [inline html/changelogs/Chronitonity-ChemBuff.yml]: double "" aaaaaaaaaaa plus no ending "s

**Comment by Chronitonity** [comment]
Holy fuck lmao

**Comment by TDcoolguy300** [comment]
:td:

File: html/changelogs/Chron - Chembuff.yml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+author: Chronitonity
+
+delete-after: True
+
+changes: 
+    - tweak: "All reagent containers (beakers, glasses, buckets ect) now is done by clicking them in your hands, just like how spray bottles work.
```
Comment (Carbonhell) [inline html/changelogs/Chron - Chembuff.yml]: still lacking the final "

**Comment by danhibiki** [comment]
Totally for this

**Comment by Foosili** [comment]
Yes. Yes. Yes. :+1:

**Comment by Nebyab** [comment]
I like right click verbs

**Comment by Chronitonity** [comment]
I dunno if your serious @nebyab but right click verbs are unintuitive and end up not getting used just because of how shit they are.

Not to mention if you're an admin you have 100~ to sort through if you wanna change something.

**Pull request merged**